SiennaOpenAPIModels.jl/src/dbinterface/db_definition.jl

```diff
@@ -83,7 +83,7 @@ const TABLE_SCHEMAS = Dict(
     ),
     "hydro_reservoir" => Tables.Schema(["id", "name"], [Int64, String]),
     "hydro_reservoir_connections" =>
-        Tables.Schema(["turbine_id", "reservoir_id"], [Int64, Int64]),
+        Tables.Schema(["source_id", "sink_id"], [Int64, Int64]),
```

Comment:
Why are we naming it `sink_id`

Reply:
It turns out we have connections between reservoirs<->reservoirs, and reservoirs<->turbines.